docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,285 @@
+
+
+    ## Rule Logic:
+  - create a value to track the rule status: `rule_status = "FAIL"`
+  - create a variable to track any notes: `rule_note = """`
+  - there should be one SHW Equipment for this space type: `if len(shw_types_and_use_dict[shw_bat]["SHWHeatingEq"]) != 1:`
+    - set rule_note to "more than one ServiceWaterHeatingEquipment" `rule_note = "more than one ServiceWaterHeatingEquipment"`
+  - otherwise, do the calculations to figure out of the SWH equipment meets the requirements of the rule: `else:`
+    - get the service water heating equipment id: `service_water_heating_equipment_id = shw_equip_dict[shw_bat]["SHWHeatingEq"]`
+    - we need to find the following information about the SWH:
+      - tank_type - (ELECTRIC_RESISTANCE_INSTANTANEOUS, ELECTRIC_RESISTANCE_STORAGE, GAS_STORAGE, OTHER)
+      - swh_type - one of the
+      - input_capacity (btu/hr)
+      - storage_volume (gallons)
+      - draw_pattern (VERY_SMALL, LOW, MEDIUM, or HIGH)
+      - capacity_per_volume (btu/hr/gallon)
+    - get the service water heating system: `swh_eq = get_object_by_id(service_water_heating_equipment_id,RMD)`
+    - get the type of tank: `tank_type = swh_eq.tank.type`
+    - get the type of heater: `swh_type = swh_eq.heater_type`
+    - the type will be one of the ServiceWaterHeaterTankOptions, which are more detailed than the four types we need.  Determine our type based on the detailed type.
+    - set the type to OTHER.  If the tank_type is an identifiable type, type will be overwritten to that type: `type = "OTHER"`
+    - if the tank_type is CONSUMER_INSTANTANEOUS, COMMERCIAL_INSTANTANEOUS, or RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS, set the type to INSTANTANEOUS: `if tank_type in ["CONSUMER_INSTANTANEOUS", "COMMERCIAL_INSTANTANEOUS", "RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS"]: type = "INSTANTANEOUS"`
+    - if the tank_type is CONSUMER_STORAGE or COMMERCIAL_STORAGE, set the type to STORAGE: `if tank_type in ["CONSUMER_STORAGE", "COMMERCIAL_STORAGE"]: type = "STORAGE"`
+    - get the fuel type of the equipment: `fuel_type = shw_eq.heater_fuel_type`
+    - set the type based on tank_type and fuel type: `if fuel_type == "ELECTRICITY":`
+      - `if tank_type == "INSTANTANEOUS": type = "ELECTRIC_RESISTANCE_INSTANTANEOUS"`
+      - `elsif tank_type == "STORAGE": type = "ELECTRIC_RESISTANCE_STORAGE"`
+    - `elsif fule_type == "NATURAL_GAS":`
+      - `if tank_type == "INSTANTANEOUS": type = "GAS_INSTANTANEOUS"`
+      - `elsif tank_type == "STORAGE": type = "GAS_STORAGE"`
+    - `elsif fule_type == "FUEL_OIL":`
+      - `if tank_type == "INSTANTANEOUS": type = "OIL_INSTANTANEOUS"`
+      - `elsif tank_type == "STORAGE": type = "OIL_STORAGE"`
+    - if the draw_pattern is given, get the draw pattern: `if shw_eq.draw_pattern != nil: draw_pattern = shw_eq.draw_pattern`
+    - otherwise, determine the draw pattern based off the first_hour_rating of the shw_eq: `else:`
```

Comment:
I don't think you can determine the draw pattern from the water heater itself, correct? It has to do with usage patterns.